downloads.md

```diff
@@ -107,6 +107,28 @@ Please have a look at the [Release Notes for Flink {{ flink_release.version_shor
 
 {% endfor %}
 
+## Flink connectors
```

Comment:
You missed `downloads.zh.md`

Reply:
Left it intentionally in case changes are requested. Will sync it in before the merge.